paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodsUiExtension.kt

```diff
@@ -45,8 +46,9 @@ internal fun PaymentMethod.getLabel(resources: Resources): String? = when (type)
     else -> null
 }
 
-internal fun PaymentMethod.getLabelIcon(): Int? = when (type) {
-    PaymentMethod.Type.USBankAccount -> FinancialConnectionsR.drawable.stripe_ic_bank
+internal fun PaymentMethod.getLabelIcon(): Int? = when {
+    type == PaymentMethod.Type.USBankAccount &&
+        DefaultIsFinancialConnectionsAvailable()() -> FinancialConnectionsR.drawable.stripe_ic_bank
```

Comment:
What's going on with the double parenthesis here?

Reply:
It is invoking the operation:

```
class DefaultIsFinancialConnectionsAvailable {
    override operator fun invoke(): Boolean = ...
}
```

Reply:
Should we just move the icon instead of adding a check here?

Reply:
I like that too!